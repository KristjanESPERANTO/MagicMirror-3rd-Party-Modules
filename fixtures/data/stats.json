{
  "moduleCounter": 7,
  "modulesWithImageCounter": 3,
  "modulesWithIssuesCounter": 6,
  "issueCounter": 9,
  "lastUpdate": "2025-09-28T00:00:00.000Z",
  "repositoryHoster": {
    "github": 7
  },
  "maintainer": {
    "CFenner": 1,
    "ItayXD": 1,
    "j-hiller": 1,
    "maxbethge": 1,
    "rubinho101": 1,
    "trumpetx": 1,
    "Unclearogre": 1
  }
}
