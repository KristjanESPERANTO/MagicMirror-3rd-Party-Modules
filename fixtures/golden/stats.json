{
  "lastUpdate": "1970-01-01T00:00:00.000Z",
  "maintainer": {
    "0lek": 1,
    "0m4r": 3,
    "2hdlockness": 3,
    "73cirdan": 3,
    "AJOCHAM": 1,
    "ASteinsdoerfer": 2,
    "AdamKogut": 1,
    "AdamMoses-GitHub": 5,
    "Adman": 1,
    "AgP42": 2,
    "AlexDespain01": 1,
    "AlexMNet": 1,
    "AlexanderSalter": 1,
    "Alvinger": 3,
    "Amand-M": 1,
    "Andoramb": 1,
    "AndreaGrandieri": 1,
    "AndreasHagman": 1,
    "AndyHazz": 3,
    "Anonym-tsk": 3,
    "Arve": 1,
    "Ax-LED": 3,
    "B0neD0ct0r": 1,
    "BKeyport": 2,
    "Bangee44": 3,
    "Bee-Mar": 3,
    "BenRoe": 1,
    "BerkSmash1984": 1,
    "BigMac1999": 1,
    "Blastitt": 2,
    "BrianHepler": 5,
    "Brtrnd": 1,
    "ByteExceptionM": 1,
    "C-Deck": 1,
    "CFenner": 7,
    "Canonip": 1,
    "Captsi": 1,
    "CatoAntonsen": 2,
    "CharlesMorris78": 1,
    "ChielChiel": 1,
    "ChrisF1976": 18,
    "ChristianGeie": 1,
    "Coernel82": 3,
    "Com-Lum": 1,
    "Comf0rTS1997": 1,
    "CptMeetKat": 2,
    "DMailMan": 1,
    "Daniel1145": 1,
    "DanielHabenicht": 1,
    "DavidLSmyth": 1,
    "Dennis-Rosenbaum": 3,
    "Dentrax": 1,
    "DimmanT": 1,
    "Dobherrmann": 1,
    "DreamyChloe": 2,
    "Duske": 1,
    "DustinBryant": 1,
    "DzMassi": 1,
    "E3V3A": 2,
    "EbenKouao": 2,
    "El-Chloro": 1,
    "ElliAndDad": 1,
    "EnderFlop": 1,
    "Eunanibus": 2,
    "EwwYumYuck": 1,
    "FabianNiehaus89": 1,
    "Fabrizz": 2,
    "FalseIlyu": 1,
    "Fifteen15Studios": 3,
    "FlatPepsi17": 1,
    "Fneves7": 1,
    "GHLasse": 1,
    "GaryLChew": 1,
    "Gismo150": 1,
    "Harishcchadichal": 1,
    "Hcreak": 1,
    "HeikoGr": 1,
    "Helpi90": 3,
    "HolmstromL": 1,
    "Iketaki": 1,
    "Infinitte": 1,
    "IoT-Coder": 1,
    "Isaac-the-Man": 1,
    "ItayXD": 1,
    "ItsCrotix": 1,
    "ItsMeBrille": 4,
    "J0n4e": 1,
    "JFWenisch": 1,
    "Jacopo1891": 2,
    "Jan-Bart": 1,
    "Jargendas": 1,
    "JasonVonKrueger": 1,
    "Jilano5": 1,
    "JockeyDoe": 1,
    "JonasGoebel": 1,
    "JonasTriki": 2,
    "JonathanAndersonPE": 1,
    "Jopyth": 2,
    "JosefJantzen": 1,
    "JulianEgbert": 1,
    "JulienPeloton": 1,
    "Jupter1": 1,
    "KayosPi": 1,
    "Kiina": 1,
    "KirAsh4": 2,
    "Klettner": 2,
    "Klizzy": 1,
    "KoblerS": 1,
    "Kreshnik": 1,
    "KristjanESPERANTO": 9,
    "Lavve": 4,
    "LeftBrainCreated": 1,
    "Legion2": 1,
    "Linas1234": 1,
    "LindsayCole": 1,
    "LoneWanderer-GH": 1,
    "LukasKlose": 1,
    "LukasWestholt": 1,
    "LukeSkywalker92": 2,
    "MMM-CalendarExt2": 4,
    "MMRIZE": 22,
    "MStoltzfus": 1,
    "MagicMirrorModules": 1,
    "MajorChris1983": 1,
    "MarcLandis": 5,
    "MartinGris": 1,
    "MartinLoidolt": 1,
    "MattLugar": 1,
    "MatthiPi": 1,
    "Matzefication": 1,
    "Medve01": 1,
    "Menturan": 2,
    "MichMich": 3,
    "MichiMich": 1,
    "MikeBishop": 2,
    "MitchSS": 1,
    "Mong-ni": 1,
    "Monkeyanator": 1,
    "NateBrownProjects": 1,
    "NateDee": 1,
    "NicholasRasi": 1,
    "NickEngmann": 1,
    "Nickiroo": 1,
    "NikolasRupp": 1,
    "Nikro": 1,
    "No3x": 1,
    "NolanKingdon": 2,
    "O5ten": 1,
    "OrangeHoopla": 1,
    "OscarVsp": 1,
    "PabloDons": 1,
    "PakshalGada": 1,
    "PalatinCoder": 1,
    "Pascal-nie": 1,
    "PatriceG": 1,
    "PavlenkoM": 2,
    "Pejjo": 1,
    "PhantomOffKanagawa": 1,
    "PierreGode": 6,
    "Pip1405": 1,
    "PjTewkesbury": 2,
    "PoOwAa": 1,
    "Privacywonk": 1,
    "ProfKP": 1,
    "Protonosgit": 1,
    "PtrBld": 1,
    "QNimbus": 1,
    "Rallymen007": 1,
    "RaspiManu": 1,
    "RaxoCoding": 1,
    "RayTCosgrove": 1,
    "RedNax67": 3,
    "RineshRamadhin": 1,
    "RpDp-git": 1,
    "Ruud14": 1,
    "SAR71": 1,
    "SBrendan": 1,
    "SLFaleiro": 1,
    "SVendittelli": 2,
    "SWhardfish": 1,
    "SageBrushes": 1,
    "SalekurPolas": 1,
    "SaltyRiver": 1,
    "Si-Hill": 1,
    "Sickboy78": 1,
    "SikandAlex": 1,
    "Simon089": 1,
    "Sispheor": 1,
    "Sjohn21": 1,
    "SkogDev": 1,
    "Smiche": 1,
    "SmolinskiP": 1,
    "Snille": 5,
    "Springvar": 1,
    "StefanNienhuis": 1,
    "TemianMihai": 1,
    "TeraTech": 1,
    "ThePirateWhoSmellsOfSunflowers": 1,
    "TheVieiraAkA": 1,
    "Thlb": 3,
    "TobbenTM": 1,
    "Tom-Hirschberger": 15,
    "Tomadelostacos": 1,
    "TouaregWarrior": 5,
    "Travelbacon": 2,
    "Tueti": 1,
    "Tuxdiver": 1,
    "Txukie": 2,
    "Tydyrium": 1,
    "Unclearogre": 3,
    "Veldrovive": 3,
    "Wanztwurst": 1,
    "WilliamFitzjohn": 1,
    "WouterEekhout": 1,
    "XBCreepinJesus": 3,
    "Xanthus1": 1,
    "Xioner19": 1,
    "Xkhi": 1,
    "XppaiCyberr": 7,
    "Yours3lf": 1,
    "Yr": 1,
    "ZakarFin": 1,
    "Zeprakadebra": 1,
    "ZeroTwoWeeb": 1,
    "aSouchereau": 1,
    "aaron64": 1,
    "aduyng": 2,
    "ahmedwab": 2,
    "ajones": 1,
    "alanshen111": 1,
    "alberttwong": 1,
    "aleckeller": 1,
    "alex-grindrod": 1,
    "alexBeuth": 1,
    "alexandreLavenant": 1,
    "alexyak": 2,
    "alltopafi": 1,
    "alonsourbano": 1,
    "ambarusa": 1,
    "amcolash": 1,
    "amitchone": 1,
    "andersarstrand": 1,
    "andreakn": 1,
    "andrecarlucci": 1,
    "anthpham": 1,
    "appdevelopmentandsuch": 1,
    "arirobinson": 1,
    "armybean": 1,
    "arnaudsnomsed": 1,
    "arthurgarzajr": 1,
    "artificialai223": 1,
    "aserramonner": 1,
    "ashishtank": 1,
    "asimhsidd": 3,
    "avianto": 2,
    "awitwicki": 2,
    "b-reich": 1,
    "bacherd": 1,
    "balassy": 7,
    "balharrie": 1,
    "barnosch": 1,
    "basknol": 2,
    "basti0001": 1,
    "bendardenne": 2,
    "benjaminflessner": 1,
    "berryerlouis": 1,
    "bertieuk": 1,
    "bibaldo": 5,
    "bigfoott": 1,
    "bitte-ein-bit": 1,
    "bittiez": 1,
    "blighter": 1,
    "blixhavn": 2,
    "bnitkin": 1,
    "boazarad": 1,
    "bogenpirat": 2,
    "bogomips": 1,
    "brentpabst": 1,
    "brssaricicek": 1,
    "brucetony": 1,
    "bryngo": 1,
    "btastic": 3,
    "btoconnor": 2,
    "buge": 1,
    "bunnymc": 1,
    "bureus": 3,
    "buxxi": 6,
    "byronxlg": 1,
    "c-klinger": 1,
    "cSharpBastard": 1,
    "calonmerc": 1,
    "carstendirks": 1,
    "cbrooker": 1,
    "cgillinger": 2,
    "chicohernando": 2,
    "choffmann": 3,
    "christopherpthomas": 1,
    "ckoutavas": 2,
    "clegallic": 1,
    "closingin": 1,
    "coderpussy": 3,
    "codetheweb": 1,
    "codingis-life": 1,
    "cokceken": 1,
    "consus2903": 1,
    "cowboysdude": 4,
    "crice009": 1,
    "crimsonclyde": 2,
    "crisvdn": 1,
    "croxis": 2,
    "cslev": 1,
    "cure": 1,
    "cybex-dev": 3,
    "dWoolridge": 1,
    "da4throux": 3,
    "dancj": 1,
    "daniel-windsor": 1,
    "danito": 1,
    "darickc": 1,
    "daterrell": 3,
    "dathbe": 6,
    "daxiang28": 1,
    "dbeltjr": 2,
    "dcwestra": 2,
    "deg0nz": 4,
    "dekinet": 3,
    "delightedCrow": 1,
    "dennisklad": 1,
    "denverquane": 1,
    "derekn": 1,
    "desertblade": 1,
    "devduisburg": 1,
    "dfanica": 1,
    "dgburr": 2,
    "dientuki": 1,
    "dierckxtom": 1,
    "dimitrystd": 1,
    "diyeverythingvn": 1,
    "djerik": 1,
    "djey47": 2,
    "dmcinnes": 1,
    "dnmmrdr1": 1,
    "doctorfree": 4,
    "dolanmiu": 1,
    "dr4ke616": 1,
    "drtorchwood": 1,
    "drventure": 2,
    "dsouzadrian": 1,
    "dwoolridge": 1,
    "eckonator": 5,
    "edward-shen": 3,
    "eerotal": 1,
    "efritz09": 1,
    "elaniobro": 2,
    "emanuele-albini": 1,
    "enarciso": 1,
    "eouia": 5,
    "erchenger": 1,
    "eric-h-st": 1,
    "erikmohn": 1,
    "ethanpartida": 1,
    "evanhsu": 1,
    "evroom": 1,
    "falafel2222": 1,
    "ferferga": 1,
    "fering3": 1,
    "fewieden": 14,
    "fhinder": 3,
    "flick116": 2,
    "flo80": 1,
    "fmandal": 1,
    "fpfuetsch": 1,
    "framboise-pi": 3,
    "frdteknikelektro": 1,
    "fruestueck": 1,
    "fry0815": 2,
    "fuerst": 1,
    "galaktor": 1,
    "gauravsacc": 1,
    "gberg927": 1,
    "gertperdZA": 1,
    "gfischershaw": 1,
    "gimdh": 1,
    "git-pesche": 1,
    "gitgitaway": 6,
    "gjonesme": 1,
    "glitch452": 4,
    "goedh452": 1,
    "gonzonia": 1,
    "gowthamravichandran": 1,
    "gpailler": 1,
    "grabenhenrich": 2,
    "gravitykillseverything": 2,
    "grenagit": 8,
    "gueguet": 2,
    "hakanmhmd": 1,
    "hansman84": 1,
    "harv-singh": 1,
    "hayseedbyte": 2,
    "hdurdle": 1,
    "hermanho": 2,
    "heskja": 1,
    "hggh": 1,
    "hoyski": 1,
    "htilburgs": 6,
    "hxreborn": 1,
    "ianmccon": 1,
    "ianperrin": 4,
    "iceship": 1,
    "idoodler": 2,
    "ilsegs": 1,
    "imperian": 1,
    "ioBroker": 1,
    "irsheep": 1,
    "istvanfedak": 1,
    "ivlovric": 2,
    "j-hiller": 1,
    "jackmeyer": 1,
    "jacquesCedric": 1,
    "jalibu": 5,
    "jancalve": 1,
    "janm84": 1,
    "jannekalliola": 2,
    "jaumebosch": 1,
    "jaylingelbach": 1,
    "jbamford": 1,
    "jboucly": 4,
    "jcktwd": 1,
    "jclarke0000": 3,
    "jclarke54": 1,
    "jdonge": 1,
    "jeroenpeters1986": 3,
    "jesmak": 3,
    "jharttech": 2,
    "jhwelch": 1,
    "jimbydude": 1,
    "jimmykh": 1,
    "jinserk": 1,
    "jirsis": 2,
    "jkschoen": 1,
    "jmwyds": 1,
    "joanaz": 2,
    "john3300": 1,
    "johnny-co": 1,
    "jojoduquartier": 1,
    "jordy-u": 1,
    "joschi27": 1,
    "jpcaldwell30": 1,
    "jrettsch": 1,
    "jschless": 1,
    "jsteel715": 1,
    "jttaylor": 1,
    "jugler": 1,
    "juliankern": 1,
    "jupadin": 7,
    "justjim1220": 7,
    "juzim": 1,
    "jwcnewton": 1,
    "kaelspencer": 1,
    "kagjerde": 3,
    "kahllenny": 1,
    "kalliope-project": 1,
    "kapsolas": 2,
    "kds54": 1,
    "kevinatown": 1,
    "khassel": 7,
    "kjb085": 1,
    "kodejak": 1,
    "kolbyjack": 2,
    "koxm": 2,
    "kstolk": 1,
    "kuitin": 1,
    "kurtern84": 3,
    "kyle-kelly": 3,
    "kymeyer": 1,
    "l0b5ter": 1,
    "l3d00m": 1,
    "late4marshmellow": 1,
    "lavolp3": 2,
    "lbartk": 1,
    "ledsi": 1,
    "lekesako": 1,
    "lindsaycole": 1,
    "linuxtuxie": 1,
    "maan75": 1,
    "mabahj": 1,
    "macd2point0": 1,
    "magic21nrw": 1,
    "malako": 1,
    "maloakes": 1,
    "marcelwinh": 1,
    "marcinomen": 1,
    "marcobiedermann": 1,
    "marcomerens": 1,
    "marska": 1,
    "martijndoes": 1,
    "martinburheimtingstad": 1,
    "martinkooij": 2,
    "masters1222": 1,
    "mathew-fleisch": 2,
    "mathewmeconry": 1,
    "matiasmtl": 1,
    "matskkolstad": 1,
    "matt-thurling": 2,
    "mattdy": 2,
    "matteodanelli": 1,
    "mattileh": 2,
    "maxbethge": 1,
    "maxenxe": 1,
    "mazim-co": 1,
    "mboskamp": 3,
    "mcintyrehh": 1,
    "mercredo": 1,
    "mezderman": 2,
    "mgarrix": 1,
    "michael5r": 4,
    "michgue": 1,
    "miguelrflor": 1,
    "mihairinzis": 1,
    "mihanich": 1,
    "mirko3000": 2,
    "mitchelltmarino": 1,
    "mjb527": 1,
    "mlbonniec": 2,
    "mmourcia": 1,
    "mmtsweng": 1,
    "mochman": 1,
    "moejetz": 1,
    "moinahmed001": 1,
    "morozgrafix": 1,
    "mosesyong": 1,
    "motey": 1,
    "mrVragec": 2,
    "mrdis": 2,
    "mrvanes": 1,
    "mscharl": 2,
    "mstarzec386": 1,
    "msteinkogler": 1,
    "mtatsuma": 2,
    "muhammedsaidckr": 1,
    "muilpp": 1,
    "mumblebaj": 12,
    "mvlbarcelos": 1,
    "mykle1": 42,
    "n-gao": 1,
    "n00b42": 1,
    "nebulx29": 2,
    "neshkoli": 2,
    "ngnijland": 4,
    "nictruong": 1,
    "nigel-daniels": 7,
    "nikobix": 3,
    "nikxif": 1,
    "nischi": 4,
    "nitpum": 1,
    "nixnuex": 2,
    "njwilliams": 3,
    "nkl-kst": 2,
    "nomis6432": 2,
    "normyx": 2,
    "nout-kleef": 1,
    "nsieg": 1,
    "nsprawling": 1,
    "oemel09": 1,
    "oenstrom": 1,
    "olexs": 2,
    "ottopaulsen": 3,
    "overflOw11": 1,
    "owenfromcanada": 1,
    "pal-thomassen": 1,
    "paphko": 2,
    "parnic": 6,
    "pbuabthong": 2,
    "pcurt": 1,
    "pelaxa": 1,
    "per-garden": 1,
    "peteyjaym": 1,
    "phpwun": 1,
    "phungorquan": 1,
    "pinsdorf": 1,
    "pixxelfriend": 1,
    "pjestico": 1,
    "polarbearjngl": 2,
    "poolis": 1,
    "prysme01": 1,
    "ptornhult": 1,
    "ptrbld": 3,
    "ptrk95": 1,
    "qistoph": 3,
    "qunu": 2,
    "rahultadak": 1,
    "ralberth": 4,
    "randomBrainstormer": 1,
    "raulbalanza": 1,
    "raywo": 2,
    "rc7": 1,
    "rcberg3": 1,
    "rcollie": 2,
    "rehne": 1,
    "reidarw": 3,
    "rejas": 1,
    "relm923": 2,
    "retroflex": 8,
    "richardfous": 2,
    "rkorell": 1,
    "robotfishe": 2,
    "ronny3050": 2,
    "roramirez": 1,
    "roufri": 1,
    "rseidt": 1,
    "rsperezn": 1,
    "rubinho101": 1,
    "ryan-d-williams": 1,
    "ryanjblajda": 1,
    "ryanzor": 1,
    "ryck": 3,
    "sabo10o29": 2,
    "sakirtemel": 1,
    "salpar": 1,
    "saml-dev": 1,
    "santi4488": 1,
    "sathyarajv": 1,
    "schlomm": 1,
    "schnibel": 1,
    "schris88": 1,
    "scottcl88": 1,
    "sdetweil": 4,
    "seb-ma": 1,
    "sebastian-ch": 1,
    "sebastianhodapp": 1,
    "sebikolon": 1,
    "seeshaughnessy": 1,
    "sejka": 1,
    "semox": 1,
    "shaneapowell": 1,
    "sharmstr": 1,
    "shbatm": 10,
    "sheyabernstein": 3,
    "shin10": 10,
    "sisimomo": 1,
    "skastenholz": 1,
    "skruglov2023": 1,
    "skuethe": 3,
    "slametps": 6,
    "slugmuffin": 1,
    "smartgic": 1,
    "sourceforge807": 1,
    "soyrubio": 1,
    "sp4ceman": 1,
    "spectroman": 3,
    "spelinski": 1,
    "spitzlbergerj": 3,
    "splattner": 1,
    "spoturdeal": 2,
    "spydersoft-consulting": 3,
    "sqall01": 1,
    "st3v0rr": 2,
    "stefanjacobs": 1,
    "stewartmckee": 1,
    "sthuber90": 1,
    "stonecrown": 1,
    "stuloh": 1,
    "supermem613": 5,
    "superuserx": 1,
    "surekap": 1,
    "syedali040205": 1,
    "szech": 1,
    "tabsl": 1,
    "tataille": 1,
    "taxilof": 1,
    "taylornoss": 1,
    "tbouron": 3,
    "tderflinger": 3,
    "teemoo7": 2,
    "thariq-shanavas": 2,
    "thekampany": 2,
    "themox": 2,
    "thesoftwarejedi": 1,
    "thess": 1,
    "thiagomides": 1,
    "thobach": 3,
    "thomaspich": 1,
    "thomo": 1,
    "tie624": 1,
    "timdows": 2,
    "timjong93": 1,
    "timodejong95": 1,
    "timophey": 1,
    "tjmnmk": 1,
    "tkrywit": 1,
    "toddsantoro": 1,
    "tomgroenwoldt": 1,
    "tomoki13": 1,
    "tonymorris-MM": 1,
    "totoluto": 3,
    "tristanburke": 1,
    "trizome": 1,
    "trumpetx": 2,
    "tschumel": 1,
    "tsoonav": 1,
    "tticehurst": 1,
    "tttooommm56": 2,
    "tylerstambaugh": 2,
    "ulrichwisser": 5,
    "uncleLem": 1,
    "uok825": 1,
    "uphillcheddar": 2,
    "usncahill": 1,
    "utzl": 1,
    "uxigene": 4,
    "valmassoi": 1,
    "vanhoekd": 1,
    "victor-paumier": 2,
    "vikramraja1995": 1,
    "vincep5": 6,
    "vita10gy": 1,
    "wathmal": 1,
    "wctimmer": 1,
    "werthdavid": 3,
    "whyjustin": 1,
    "wiedsee": 1,
    "wiesty": 4,
    "wilfullyapt": 1,
    "winstonma": 1,
    "wkra": 1,
    "wlans": 1,
    "wonderslug": 1,
    "wouter-heerwegh": 1,
    "wzooff": 1,
    "x3mEr": 1,
    "xadamxk": 5,
    "xppaicyberr": 1,
    "yashatgit": 2,
    "yawnsde": 1,
    "yedidiaklein": 1,
    "yo-less": 2,
    "yohaybn": 1,
    "yourdawi": 4,
    "ytmytm": 1,
    "yuhanx0728": 1,
    "yusinv": 1,
    "zanix": 1,
    "zerosamski": 1,
    "zhouxichun": 1,
    "zuo000": 1,
    "zzhenney": 1
  },
  "moduleCounter": 1301,
  "modulesWithImageCounter": 926,
  "modulesWithIssuesCounter": 1300,
  "repositoryHoster": {
    "bitbucket": 1,
    "github": 1288,
    "gitlab": 12
  }
}
